www/src/pages/insights.jsx

```diff
@@ -171,7 +173,16 @@ const ComponentsUsage = () => (
   </div>
 );
 
-export default function InsightsPage() {
+export default function InsightsPage({ location }) {
+  const tabs = ['/insights', '/insights?tab=projects', '/insights?tab=components'];
+  const path = `${location.pathname}${location.search}`;
+
+  const handleOnSelect = (value) => {
+    if (value !== path) {
+      navigate(value);
```

Comment:
I believe we also will need to get the current tab, if any, from the URL and navigate to it upon page load, so that if I hard refresh on `/insights?tab=components`, I will see the contents of the "Components" tab already.

Reply:
Yes, I also saw this in the preview. Locally it works fine. On preview it is broken for some reason. I thought it could be due to the forward slashes (`/insights?tab=component` changed `/insights/?tab=component`) but it is not. Currently I am studying the issue.
Also I am interested if you have any ideas why this can happen 🤔

Reply:
@adamstankiewicz I think we were able to fix it, I believe gatsby just served the default page version that was created during the build and javascript just didn't get a chance to run on first render to update selected tab based on the url (also I think gatsby doesn't like query parameters in the url...)

Manually building separate page for each tab solves the issue